openmls/src/group/tests/utils.rs

```diff
@@ -419,32 +416,19 @@ pub(crate) fn resign_message(
 #[cfg(test)]
 pub(crate) fn resign_external_commit(
     signer: &impl Signer,
-    plaintext: PublicMessage,
-    original_plaintext: &PublicMessage,
+    public_message: PublicMessage,
+    old_confirmation_tag: ConfirmationTag,
```

Comment:
We do not use anything from the `original_plaintext` despite the confirmation tag. Thus, I made that more clear.